static/js/admin/src/statusMonitor/DepositMonitor.js

```diff
@@ -105,18 +105,19 @@ define('DepositMonitor', [ 'jquery', 'jquery-ui', 'underscore', 'AbstractStatusM
 		if ("currentJob" in results) {
 			var currentJob = results.jobs[results.currentJob];
 		}
-		
+
 		if ("jobs" in results) {
 			for (var index in results.jobs) {
 				var job = results.jobs[index];
 				var jobName = job.name.substring(job.name.lastIndexOf(".") + 1);
 				
 				job["shortName"] = jobName;
-				
-				if (jobName == "IngestContentObjectsJob" && "total" in job) {
-					var completion = results.ingestedObjects? results.ingestedObjects : 0;
-					completion += " / " + job.total;
-					job["completion"] = completion;
+
+				var hasCount = job.num > 0;
```

Comment:
This section looks good.

While testing it, I was finding with so many additional progress numbers showing it was getting a bit hard to differentiate between all the numbers next to each other. Adding some text to split up the progress from the time spent helps a little. Changing the following line:
https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/fcrepo4/static/templates/admin/statusMonitor/depositMonitorJobDetails.html#L30
to put "in" between progress and time:
`<li><span title="<%- job.shortName %>"><%= job.shortName %></span> <%= job.status %> <%= job.completion %> <% if (job.time) { %> in <%= job.time %> <% } %></li>`